tools/ci-scripts/windows/Install-Boost.ps1

```diff
@@ -122,14 +122,18 @@ function Install-BoostComponent([string]$Component)
             -PackageId $packageId `
             -InstallDir $workDir `
             -PackageVersion $Version
-
+        if($Version < 1.66){
         Move-Item `
-            -Path ([System.IO.Path]::Combine($workDir, $packageId, 'lib', 'native', 'address-model-32', 'lib', '*')) `
+            -Path ([System.IO.Path]::Combine($workDir, $packageId, 'lib', 'native', '*-x32-*', 'lib', '*')) `
             -Destination $lib32Dir
 
         Move-Item `
-            -Path ([System.IO.Path]::Combine($workDir, $packageId, 'lib', 'native', 'address-model-64', 'lib', '*')) `
+            -Path ([System.IO.Path]::Combine($workDir, $packageId, 'lib', 'native', '*-x64-*', 'lib', '*')) `
```

Comment:
Under the 1.66+ model, I believe that all the files are under `lib\native` with names like `foo-x32-stuff.dll`. This copy command is trying to copy a directory that doesn't exist right now, it looks.

If I were to do this, I'd install a copy of some of the packages into a temp directory and see how they're laid out.

Reply:
I could not understand the meaning of the above suggestion, can you elaborate a bit more, on the file structure, and about .dll and .ldb files in the directory.

Reply:
I suggest that you get the Bond CMake C++ build working with Boost 1.65. Then, download the Boost 1.66 packages, extract them, and try to get the build working with those packages. You'll need to delete your `build/` directory each time you try something new so that CMake regenerates from scratch. Things I'd try:

* Pointing `$env:BOOST_LIBRARYDIR` at the `lib\native` directory without renaming any thing.
* Create an `x32` directory and move all the `*x32*` files into that directory.
* Create an `x32` directory and move all the `*x32*` files into that directory and strip the `x32` component of the file name.